docs/resources/bedrock_blueprint.md

```diff
@@ -36,6 +36,10 @@ resource "awscc_bedrock_blueprint" "example" {
       }
     }
   })
+  tags = [{
```

Comment:
please add tags to the main.tf file and re-run make docs, it should render the .md file automatically